api/src/main/java/org/openmrs/module/pihcore/config/Components.java

```diff
@@ -18,14 +18,19 @@
         public static final String ONCOLOGY = "oncology";  // program
         public static final String NCD = "ncd";  // program
         public static final String HIV = "hiv";  // program. Needs HIVProgramBundle and Haiti HIV MDS package
+        public static final String HIV_PROGRAM = "hivProgram";
+        public static final String HIV_FORMS = "hivForms";
+        public static final String HIV_INTAKE_FORM = "hivIntakeForm";
```

Comment:
Would be good to document here what each does:
* HIV_INTAKE_FORM: just Intake
* HIV_FORMS: all forms
* HIV_PROGRAM: just HIV Program
* HIV: HIV Program + All HIV Forms

... maybe even with the note that you should only ever have to include one of these per implementation (right?)  (It totally makes sense to me that HIV_FORMS should include the the intake form, so you don't have to enable both HIV_FORMS and HIV_INTAKE_FORMS, but I worry that without documentation people will set them all within a single implementation "just to safe" leading to confusing down the road).

Reply:
Oh yeah, I'd meant to do that! Thanks.